scripts/2.TCGA-process.py

```diff
@@ -232,37 +244,44 @@
 expr_df.shape
 
 
-# In[20]:
+# In[21]:
 
 # Number of patients represented in the expression dataset
 clinmat_df.query("sample_id in @expr_df.index").patient_id.nunique()
 
 
-# In[21]:
+# In[22]:
 
 # Peak at the data matrix
 expr_df.iloc[:5, :5]
 
 
+# In[23]:
+
+# Save complete expression matrix
+path = os.path.join('data', 'complete', 'expression-matrix.tsv.bz2')
+expr_df.to_csv(path, sep='\t', float_format='%.3g', compression='bz2')
```

Comment:
`float_format='%.3g'` is _probably_ ok? I was just curious as to your reasoning.

I'd say about half of the raw data entries that are not 0 have 3 decimal points, the other half has 4 (while some have many, which are mostly zeros).